SPECS/libkcapi/libkcapi.spec

```diff
@@ -209,6 +211,11 @@ rm -f                            \
   %{buildroot}%{_bindir}/sha*sum      \
   %{buildroot}%{_bindir}/sm*sum
 
+# Create hard-links to alias dracut-expected sha* bins to new kcapi-hasher bin.
+for app in %{apps_hmaccalc}; do
```

Comment:
Note: I checked dnf whatprovides ... for each apps_hmaccalc. No overlap in coreutils, openssl, or any other package.